src/main.rs

```diff
@@ -759,18 +759,14 @@ fn run() -> Result<(), failure::Error> {
         );
         let manifest = settings::toml::Manifest::new(config_path)?;
 
-        let mut host: Option<&str> = matches.value_of("host");
+        let host: Option<&str> = matches.value_of("host");
         let mut ip: Option<&str> = matches.value_of("ip");
         let mut port: Option<u16> = matches
             .value_of("port")
             .map(|p| p.parse().expect("--port expects a number"));
 
         // Check if arg not given but present in wrangler.toml
         if let Some(d) = &manifest.dev {
-            if host.is_none() && d.host.is_some() {
-                host = d.host.as_deref();
-            }
-
             if ip.is_none() && d.ip.is_some() {
                 ip = d.ip.as_deref();
             }
```

Comment:
i'd also say that flags should override toml settings, so if they pass `--port 9090` and in their `wrangler.toml` it says `[dev.port] = 1010` then `9090` should be chosen

Reply:
That's how it should be working now, `ip` is an Option taken from clap. If it's none and there exists something in the toml then it changes ip, otherwise it'll be the same as before